File: cartracker/src/main/webapp/app/model/option/Position.js
Ext.define("CarTracker.model.option.Position",
{
  extend : "Ext.data.Model",
  fields : [ {
    name : "longName",
    type : "string"
  }, {
    name : "shortName",
    type : "string"
  }, {
    name : "createDate",
    type : "date",
    dateFormat : "c"
  }, {
    name : "active",
    type : "boolean",
    defaultValue : true
  }, {
    name : "id",
    type : "int",
    useNull : true,
    convert : null
  } ],
  proxy : {
    type : "direct",
    api : {
      read : "positionOptionService.read",
      create : "positionOptionService.create",
      update : "positionOptionService.update",
      destroy : "positionOptionService.destroy"
    },
    reader : {
      root : "records"
    }
  }
});

File: cartracker/src/main/java/ch/rasc/cartracker/entity/option/Position.java
package ch.rasc.cartracker.entity.option;

import javax.persistence.Entity;

import ch.rasc.extclassgenerator.Model;

@Entity
@Model(value = "CarTracker.model.option.Position",
		readMethod = "positionOptionService.read",
		createMethod = "positionOptionService.create",
		updateMethod = "positionOptionService.update",
		destroyMethod = "positionOptionService.destroy", paging = true)
public class Position extends BaseOption {
	// nothing special
}
